.vscode/settings.json

```diff
@@ -1,4 +1,8 @@
 {
   "editor.trimAutoWhitespace": false,
   "files.trimTrailingWhitespace": false,
-}
\ No newline at end of file
+  "prettier.disableLanguages": [
+    "vue", //default
```

Comment:
Is vue necessary? It isn't used, unless it's being used as a hack of some sort.

Reply:
I just kept it because it's the default setting and didn't want to do anything other than add markdown. But I can remove it if you'd rather I do so.

Reply:
Might as well, there risk of Vue is low

Reply:
Done. Will merge once the lights turn green.